src/style/font/ttf.rs

```diff
@@ -100,10 +100,8 @@ fn load_font_data(face: FontFamily, style: FontStyle) -> FontResult<Font<'static
         };
         // TODO: font-kit actually have rasterizer, so consider remove dependency for rusttype as
         // well
-        let result = FontCollection::from_bytes(Into::<SharedBytes>::into(data))
-            .map_err(|err| FontError::FontLoadError(Arc::new(err)))?
-            .font_at(id.max(0) as usize)
-            .map_err(|err| FontError::FontLoadError(Arc::new(err)));
+        let result = Font::try_from_vec_and_index(data, id.max(0))
```

Comment:
Seems rusttype 0.9 doesn't have font loading error information any more, just wondering is there any way to distinguish the different kind of loading error. For example, "UnrecognizedFormat" vs "CollectionIndexOutOfBounds" https://docs.rs/rusttype/0.8.3/rusttype/enum.Error.html

Reply:
`rusttype` uses [ttf-parser](https://github.com/RazrFalcon/ttf-parser) now, which doesn't report any errors. Mainly because there are not much to tell. The API is designed around the idea that each method can produce only one kind of error. Therefore we are using `Option`.

For example, we can return an error saying that the CFF table has an outdated type1 format, but it doesn't really tell anything to a caller since you have to know all the TrueType nuances to reason about this.